<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAberturaCaixaPdv</class>
 <widget class="QWidget" name="FormAberturaCaixaPdv">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2{

	background-color: rgb(0, 0, 0,200);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);

text-transform: uppercase;
border:4px solid blue;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QFrame#frame{
color:red;
border: none;
background-color:#2408D4;}
QLabel{
color:white;
border: none;
background:none;
	font: 20pt &quot;Arial&quot;;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="LB_abertura">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>abertura caixa</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_11">
           <property name="styleSheet">
            <string notr="true">QFrame
{background: #eaeaea;
border: none
}
QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase;
border:none
}
QLineEdit:Focus {
border: 1px solid red;
}

QLabel{
text-transform: uppercase;
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLineEdit" name="tx_Descricaixa">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>251</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>descri√ßao</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>70</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>valor</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="db_valorCaixa">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>100</y>
              <width>171</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
border:none;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
           <widget class="QFrame" name="frame_12">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>490</y>
              <width>471</width>
              <height>51</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
QFrame{
border:none
}
QPushButton{
	text-align:center;
	color:#fff;
	border:0px;
	
	font-size:20px;
	cursor: pointer;
	background-color:red;
padding:5px 15px;
}
QPushButton:hover{
background-color:#3668ff;
}

QPushButton#bt_confirma{
	text-align:center;
	color:#fff;
	border:0px;
	
	font-size:20px;
	cursor: pointer;
	background-color:#538fff;
padding:5px 15px;
}
QPushButton#bt_confirma:hover{
background-color:#3668ff;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QPushButton" name="bt_confirma">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::NoContextMenu</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>salvar</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>75</width>
                 <height>35</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="bt_sairlancamento">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string>sair</string>
               </property>
               <property name="shortcut">
                <string>Esc</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QDateEdit" name="dt_incluirdata">
            <property name="geometry">
             <rect>
              <x>290</x>
              <y>30</y>
              <width>151</width>
              <height>31</height>
             </rect>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QDateEdit {
	border: 2px solid lightgray;
	border-radius: 5px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 255, 255);
}
 
</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>290</x>
              <y>10</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>data</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cb_incluirTipodocumento">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>151</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:1px solid;</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>121</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>tipo PAgamento</string>
            </property>
           </widget>
           <widget class="QTableWidget" name="tab_caixa">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>150</y>
              <width>481</width>
              <height>341</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableView{
background: #F1F1F1;
color: black;
font-weight: bold;
font-size: 16px;
border:1px solid blakc;
	
	selection-color: rgb(255, 0, 0);
	selection-background-color: rgb(85, 170, 255);
}
QHeaderView:section{
background: #FFF;

font-size: 13px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: #797979;
border:none;
border-bottom: 2px solid black;
border-top:2px solid black;
padding:5px 0px;
text-transform: uppercase
}


</string>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>tipo doc</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>entrada</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>saida</string>
             </property>
            </column>
           </widget>
           <widget class="QCommandLinkButton" name="commandLinkButton">
            <property name="geometry">
             <rect>
              <x>390</x>
              <y>100</y>
              <width>41</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
background:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tx_Descricaixa</tabstop>
  <tabstop>cb_incluirTipodocumento</tabstop>
  <tabstop>db_valorCaixa</tabstop>
  <tabstop>bt_sairlancamento</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bt_sairlancamento</sender>
   <signal>clicked()</signal>
   <receiver>FormAberturaCaixaPdv</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
